packages/frontend/src/components/ProjectConnection/DbtSettingsForm.tsx

```diff
@@ -228,6 +228,14 @@ const DbtSettingsForm: FC<DbtSettingsFormProps> = ({
                             name="Advanced"
                             isOpen={isAdvancedSettingsOpen}
                         >
+                            <Stack>
```

Comment:
Couple of changes:
- hide this from dbt cloud option
- render inside existing stack element
- update placeholder to not have `--select` 
- add description with link to dbt docs

Reply:
![image](https://github.com/user-attachments/assets/718c5f75-90ff-45cb-a7f7-b66a283e2da8)